src/main/resources/static/css/_index.css

```diff
@@ -2,16 +2,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@700&display=swap');
 
-/* FONT */
-
-/*
-P.S.
-Using fibonacci series for font size and line height.
-I thing it looks really good.
-Joe
```

Comment:
I agree, discussion doesn't belong here, and should be removed. This may have been added from another code review.